sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateOperation.cs

```diff
@@ -261,5 +269,36 @@ public virtual async Task DeleteAsync(CancellationToken cancellationToken = defa
 
             Properties = response;
         }
+
+        private class UpdateStatusActivity : IDisposable
+        {
+            private readonly CertificateOperation _operation;
+            private readonly long _start;
+
+            public UpdateStatusActivity(CertificateOperation operation)
+            {
+                _operation = operation;
+
+                if (EventSource.IsEnabled())
+                {
+                    EventSource.BeginUpdateStatus(_operation.Properties);
+
+                    _start = Stopwatch.GetTimestamp();
```

Comment:
Do we need the StopWatch if we have the timestamps from `BeginUpdateStatus` and `EndUpdateStatus`?

Reply:
That's a good point. If a listener was interested, they could print out the timestamps or even calculate it themselves. I figured this was a convenient and relatively cheap way of getting elapsed times. @pakrym what do you think? Worth keeping? I know Azore.Core does this on 1 or 2 events.

Also, I realized last night a source could be enabled or disabled in the middle, so if I keep this I should always get the start time (relatively cheap) but can optimize out the double calculation if not enabled.

Reply:
As far as I understand these events are added to diagnose possible test failures and will be removed afterward. I'm fine with whatever helps you diagnose the issue.

Reply:
Let's pretend/assume we keep them in. Does that change your mind? If I can't diagnose why these occasionally fail before GA, I might be inclined to leave them in.

I don't actually need the elapsed time so I'll just remove it.

Reply:
Let's pretend/assume we keep them in. Does that change your mind? If I can't diagnose why these occasionally fail before GA, I might be inclined to leave them in.

If we keep them forever it's very inconsistent to have it only for one operation and not the others. I'm fine leaving it for GA but if the plan is to keep it long term we need a better cross-cutting design.

Reply:
I just removed it. It really wasn't necessary for timing anyway. Was just a thought. I agree a consistent solution is warranted. You have it for 1 task (general response, I believe?) but in that case it makes sense: easily determining how long a response took from a request seems like something very common - common enough not to make people diff timestamps themselves (especially with concurrent requests and having to find the associated request). Some guideline based on that logic might be a good start.

Here, it was just info. I did it because I could - not really any more reason than that. :)